pkg/cmd/cli/cmd/debug.go

```diff
@@ -444,6 +444,12 @@ func (o *DebugOptions) Debug() error {
 		case err != nil:
 			return err
 		default:
+			// TODO this doesn't do us much good for remote debugging sessions, but until we get a local port
+			// set up to proxy, this is what we've got.
+			if podWithStatus, ok := containerRunningEvent.Object.(*kapi.Pod); ok {
+				fmt.Fprintf(o.Attach.Err, "Pod IP: %s\n", podWithStatus.Status.PodIP)
```

Comment:
> Why print to the error output?

It's where the rest are going: https://github.com/openshift/origin/pull/8724/files#diff-79c5f976ae3273934daeada9309b517dR396